non-pinned.json

```diff
@@ -13,6 +13,9 @@
   "prConcurrentLimit": 0,
   "prHourlyLimit": 0,
   "branchConcurrentLimit": 0,
+  "minimumReleaseAge": "7 days",
+  "internalChecksFilter": "strict",
+  "prCreation": "not-pending",
```

Comment:
❓ This is pretty hot and I didn't know it existed. I looked into this and if we used `status-success` and we let Renovate through check runs it will be really powerful, but ... if that merged today it would stop all PR creations in our check run-ed repos right? This is fine as-is, but maybe it's a callout for a later task to move to my suggestion.